app/views/games/show.html.erb

```diff
@@ -1,14 +1,45 @@
+<script>
+  $(function() {
+    $('.drag').draggable({
+      revert: "invalid"
+    });
+
+    $('.drop').droppable({
+      drop: function(event, ui){
+        var desiredUrl = $(ui.draggable.context.innerHTML).attr("href");
+        var x = $(this).data('square-x');
+        var y = $(this).data('square-y');
+        console.log(x, y);
+        var piece = { piece: {
+          horizontal_position: x,
+          vertical_position: y
+        } };
+        $(ui.draggable).detach().css({top: 0,left: 0}).appendTo(this);
+        $.ajax({
+          type: 'PUT',
+          url: desiredUrl,
+          data: piece,
+          dataType: 'json'
+        });
```

Comment:
I still don't know how to fix the fly back issue for invalid moves... somehow the javascript needs to talk to the move_to method in the controller and then only stick to the new spot if it passes validation. I have no idea how to do that.
For now add this so the page reloads after each move:
```
          dataType: 'json'
        }).done(function(){
          window.location.reload();
        });
      }
```